src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -357,8 +359,12 @@ class DeployTask extends AbstractServerTask {
         File parentProjectDir = new File(task.getProject().getRootProject().rootDir.getAbsolutePath())
         for (File dep: deps) {
             String dependencyProjectName = getProjectPath(parentProjectDir, dep)
-            String projectDependencyString = "project ':" + dependencyProjectName + "'"
-            Project siblingProject = project.getRootProject().findProject(dependencyProjectName)
+            String projectDependencyString
+            Project siblingProject
+            if (dependencyProjectName != null && !dependencyProjectName.isEmpty()) {
+                projectDependencyString = "project ':" + dependencyProjectName + "'"
+                siblingProject = project.getRootProject().findProject(dependencyProjectName)
+            }
```

Comment:
Question: Now that `projectDependencyString` can be null on the next line, the `isCurrentProject` would equal false in that case. This would cause the code to go into the final `else` statement. Is that the expected behavior?

Reply:
Yes, that is expected. There was an issue where the non project dependencies were getting handled as project dependencies so I added the project check.